corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -70,7 +70,7 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None):
         vellum_plugins.append("commtrack")
     if toggles.VELLUM_SAVE_TO_CASE.enabled(domain):
         vellum_plugins.append("saveToCase")
-    if toggles.VELLUM_EXPERIMENTAL_UI.enabled(domain) and module and module.case_type:
```

Comment:
We're only showing the form if it uses "updates or closes a case" in case management. User case isn't displayed right now